.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release (optional)'
+        required: false
+        type: string
+
+concurrency:
+  group: release-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+  pull-requests: read
+  statuses: write
+
+jobs:
+  weekly:
+    name: Weekly
```

Comment:
This is not weekly anymore, right?

Reply:
Goot catch! I renamed it to `continuous`.